Documentation/dev-guide/interacting_v3.md

```diff
@@ -267,14 +314,29 @@ bar
 PUT
 foo
 bar_new
+```
 
+```bash
 # watch for changes on key `foo` since revision 3
 $ etcdctl watch --rev=3 foo
 PUT
 foo
 bar_new
 ```
 
+Here is an example to watch only from the last historical change:
+
+```bash
```

Comment:
@xiang90 Actually the prev-kv flag gives out the modified key value along with its last key value. So i had framed it like that. Any suggestions to help me improve on this?